yourls/content.md

```diff
@@ -10,29 +10,42 @@ YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener
 
 ## Start a `%%IMAGE%%` server instance
 
-```console
-$ docker run --name some-%%REPO%% --link some-mysql:mysql \
-    -e YOURLS_SITE="https://example.com" \
-    -e YOURLS_USER="example_username" \
-    -e YOURLS_PASS="example_password" \
-    -d %%IMAGE%%
+```bash
+docker run \
+    --name some-%%REPO%% \
+    --detach \
+    --link some-mysql:mysql \
+    --env YOURLS_SITE="https://example.com" \
+    --env YOURLS_USER="example_username" \
+    --env YOURLS_PASS="example_password" \
+    %%IMAGE%%
 ```
 
-The YOURLS instance accepts a number of environment variables for configuration, see *Environment Variables* section below.
+The YOURLS instance accepts a number of environment variables for configuration, see [*Environment Variables*](#environment-variables) section below.
```

Comment:
Unfortunately, the actual target of this Markdown is the image description on Docker Hub, and inter-doc anchor references like this haven't consistently worked on Docker Hub (hence the cagey "see below" language).